source/config/configSpec.py

```diff
@@ -87,6 +87,7 @@
 		guessObjectPositionInformationWhenUnavailable = boolean(default=false)
 		reportTooltips = boolean(default=false)
 		reportHelpBalloons = boolean(default=true)
+		reportUIANotifications = option("always", "focusedApp", "off", default="always")

```

Comment:
I think we should keep the default on focused app, as it is now:

```suggestion
		reportUIANotifications = option("always", "focusedApp", "off", default="focusedApp")
```